frontend/scss/organisms/_o-custom-tours-viewer.scss

```diff
@@ -628,11 +570,17 @@
 }
 
 .aic-ct-list-item__artwork-img-container {
+  display: flex;
+  justify-content: center;
   margin-bottom: 12px;
+  max-height: 500px;
```

Comment:
I think you might be able to use object-fit and object-position to achieve this effect without having to set the max-height in both places.

e.g.

```
.aic-ct-list-item__artwork-img-container {
 ...
  img {
      object-fit: contain;
      object-position: center;
  }
}